Source/WebCore/dom/TreeScope.h

```diff
@@ -33,6 +33,10 @@
 #include <wtf/Vector.h>
 #include <wtf/text/AtomString.h>
 
+namespace JSC {
+class JSValue;
+} // namespace JSC
```

Comment:
I typically would not add a comment for the end of a tiny namespace section like this.

Reply:
I'll remove it before landing.